src/gesture/touchGestures.js

```diff
@@ -57,6 +57,7 @@ module.exports = {
 			var g = this.makeGesture('gesturestart', e, {vector: this.gesture, scale: 1, rotation: 0});
 			dispatcher.dispatch(g);
 		}
+		e.preventDefault();
```

Comment:
Yes @aarontam , it creates problem in scrollers, I was working on the fix for that. As of now its not a proper fix. But I felt working with mouse events in touch mode was not proper behavior. Anyway I 'll  try to find a different solution.